modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
@wing328 I am not sure about priorities - is it more important to have working client out of the box, or having more flexible one? 
Should we add client or httplug+client dependencies by default?
Is enough to provide generated apis classes calling `send($request)` on `ApiClientInterface`, and expect users to choose client library and select/create adapter?
Or leave no-client version as default and add --library=httplug?